bin/install/legacy/install-legacy-dcos-cli.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -o errexit -o pipefail
+
+usage()
+{ # Show usage information.
+  echo "$(basename "$(test -L "$0" && readlink "$0" || echo "$0")") <installation-path> <dcos-url> [--add-path yes/no]"
+}
+
+post_install_message()
+{
+    echo 'Finished installing and configuring DCOS CLI.'
+    echo ''
+    echo 'Run this command to set up your environment and to get started:'
+    echo "source $1 && dcos help"
+}
+
+RC_NAME=""
+
+write_to_profile()
+{
+    echo "" >> ~/"$2";
+    echo "# path to the DCOS CLI binary" >> ~/"$2";
+    echo "if [[ \"\$PATH\" != *\"$1\"* ]];" >> ~/"$2";
+    echo "  then export PATH=\$PATH:$1;" >> ~/"$2";
+    echo "fi" >> ~/"$2";
+}
+
+add_dcos_path_to_profile()
+{
+    UNAME=`uname`
+    case "$UNAME" in
+        Linux ) RC_NAME=".bashrc";;
+        Darwin ) RC_NAME=".bash_profile";;
+        CYGWIN* ) RC_NAME=".bashrc";;
+        MINGW* ) RC_NAME=".profile";;
+        * ) RC_NAME=".bashrc";;
+    esac
+    write_to_profile "$1" "$RC_NAME"
+}
+
+prompt_add_dcos_path_to_profile()
+{
+    while true; do
+        echo ""
+        read -p "Modify your bash profile to add DCOS to your PATH? [yes/no] " ANSWER
+        echo ""
+        case "$ANSWER" in
+            [Yy]* ) add_dcos_path_to_profile "$1"; break;;
+            [Nn]* ) break;;
+            * ) echo "Please answer yes or no.";;
+        esac
+    done
+}
+
+check_pip_version()
+{
+    PIP_INFO=$(pip -V);
+    REGEX="([0-9]+)\.([0-9]+)";
+    [[ $PIP_INFO =~ $REGEX ]];
+    MAJOR_PIP_VERSION="${BASH_REMATCH[1]}";
+    MINOR_PIP_VERSION="${BASH_REMATCH[2]}";
+    if [ "$MAJOR_PIP_VERSION" -lt 1 ] || ([ "$MAJOR_PIP_VERSION" -eq 1 ] && [ "$MINOR_PIP_VERSION" -le 4 ]);
+        then echo "Pip version must be greater than 1.4. Aborting.";
+        exit 1;
+    fi
+}
+
+if [ "$#" -lt 2 ]; then
+  usage;
+  exit 1;
+fi
+
+check_pip_version;
+
+ARGS=( "$@" );
+
+VIRTUAL_ENV_PATH=$(python -c "import os; print(os.path.realpath('"${ARGS[0]}"'))")
+if [[ $VIRTUAL_ENV_PATH =~ \  ]];
+	then echo "Spaces are not permitted in the installation path. Please try again with another path.";
+    exit 1;
+fi
+DCOS_URL=${ARGS[1]}
+
+command -v virtualenv >/dev/null 2>&1 || { echo "Cannot find virtualenv. You may need to install it by following the documentation at https://docs.mesosphere.com/install/cli/#linux. Aborting."; exit 1; }
+
+VIRTUALENV_VERSION=$(virtualenv --version)
+VERSION_REGEX="s#[^0-9]*\([0-9]*\)[.]\([0-9]*\)[.]\([0-9]*\)\([0-9A-Za-z-]*\)#\1#"
+
+eval MAJOR=`echo $VIRTUALENV_VERSION | sed -e $VERSION_REGEX`
+if [ $MAJOR -lt 12 ];
+	then echo "Virtualenv version must be 12 or greater. Aborting.";
+	exit 1;
+fi
+
+echo "Installing DCOS CLI from PyPI...";
+echo "";
+
+# Let's first setup a virtualenv: we are assuming that the path is absolute
+mkdir -p "$VIRTUAL_ENV_PATH"
+virtualenv "$VIRTUAL_ENV_PATH"
+
+# Install the DCOS CLI package, using version if set
+if [ -z "$DCOS_CLI_VERSION" ]; then
+    "$VIRTUAL_ENV_PATH/bin/pip" install --quiet "dcoscli"
```

Comment:
This is not correct. It should be something like `"dcoscli<0.4.0"`. Let's make sure we test this when we release `0.4.0`. File an issue or something so we don't forget.